packages/react-widgets/src/models/utils.js

```diff
@@ -0,0 +1,42 @@
+import { AggregationTypes, _filtersToSQL } from '@carto/react-core';
+
+export function formatTableNameWithFilters(props) {
```

Comment:
I'm not completely familiar with the organization of the code between the different packages, but the functions declared in this file are quite low-level SQL manipulations. Wouldn't they be better stored in `react-core` alongside other SQL utils?

Reply:
But in this case Benjamin queries are very tied to widgets themselves, really not reusable in other contexts.

Reply:
I agree with Victor, at least in a first iteration I think these kind of stuff should be with the widget.